### PR TITLE
EN-8854: get all key-value pairs for an account API endpoint

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Implemented the `address/{address}/keys` endpoint that will return a map containing all keys and their values for an account. The value is represented in hexadecimal encoding.

Testing procedure:
-start a testnet using the corresponding proxy's branch
-find an address that must have saved something inside it's key-value storage (for example, an address that interacted with an ESDT token) and make an API call to `proxy-url/address/erd1.../keys`. it should return a map containing all its keys

### CODE REVIEW

**Review by miiu96 — CHANGES_REQUESTED** [review]
Please add unit tests to increase coverage

**Review by SebastianMarian — APPROVED** [review]

**Review by SebastianMarian — DISMISSED** [review]

File: node/node.go [hunk 1]
```diff
@@ -440,6 +440,41 @@ func (n *Node) GetUsername(address string) (string, error) {
...
+
+	ctx := context.Background()
+	chLeaves, err := userAccount.DataTrie().GetAllLeavesOnChannel(rootHash, ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	mapToReturn := make(map[string]string)
+	for leaf := range chLeaves {
+		mapToReturn[string(leaf.Key())] = hex.EncodeToString(leaf.Value())
```
Comment (sasurobert) [inline node/node.go]: hex.encode the leaf.Key

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: api/address/routes.go [hunk 2]
```diff
@@ -31,6 +32,7 @@ type FacadeHandler interface {
 	GetCode(account state.UserAccountHandler) []byte
 	GetESDTBalance(address string, key string) (string, string, error)
 	GetAllESDTTokens(address string) ([]string, error)
+	GetKeyValuePairs(address string) (map[string]string, error)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply api/address/routes.go:35]: I will leave it like this so we'll have all of the keys and values hex encoded

File: node/node.go (hunk 1 above)
Comment (bogdan-rosianu) [reply node/node.go]: you reviewed an outdated code. it is hex encoded now

**Review by miiu96 — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by mateigraura — APPROVED** [review]
system test passed

**Review by sasurobert — APPROVED** [review]

**Pull request merged**